test/StripeResource.spec.js

```diff
@@ -557,38 +657,96 @@ describe('StripeResource', () => {
           }
         );
       });
+
+      it('invokes the callback with successful results', (done) => {
+        const returnedCharge = {
+          id: 'ch_123',
+        };
+        return utils.getTestServerStripe(
+          {},
+          (req, res) => {
+            res.write(JSON.stringify(returnedCharge));
+            res.end();
+          },
+          (err, stripe, closeServer) => {
+            if (err) {
+              return done(err);
+            }
+            stripe.charges.create(options.data, (err, result) => {
+              expect(result).to.deep.equal(returnedCharge);
+              closeServer();
+              done();
+            });
+          }
+        );
+      });
+
+      it('returns successful results to await', (done) => {
+        const returnedCharge = {
+          id: 'ch_123',
+        };
+        return utils.getTestServerStripe(
+          {},
+          (req, res) => {
+            res.write(JSON.stringify(returnedCharge));
+            res.end();
+          },
+          async (err, stripe, closeServer) => {
+            if (err) {
+              return done(err);
+            }
+            const result = await stripe.charges.create(options.data);
```

Comment:
I think this is missing a `.catch` so if stripe.charges.create fails the error won't be properly propagated to the test runner

Reply:
Done.